rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -191,7 +191,18 @@ public AccessIdRepresentationModel searchGroupByDn(final String name) {
 
     final AndFilter andFilter = new AndFilter();
     andFilter.and(new EqualsFilter(getGroupSearchFilterName(), getGroupSearchFilterValue()));
-    andFilter.and(new LikeFilter(getGroupsOfUser(), "*" + accessId + "*"));
+    final OrFilter orFilter = new OrFilter();
+    orFilter.or(new EqualsFilter(getGroupsOfUser(), accessId));
+    orFilter.or(
+        new EqualsFilter(
+            getGroupsOfUser(),
+            LdapNameBuilder.newInstance()
+                .add(getBaseDn())
+                .add(getUserSearchBase())
+                .add("uid", accessId)
```

Comment:
what happens if the accessId is already a full dn? Do we have a test for this case?

Reply:
It could be either a full dn, then "orFilter.or(new EqualsFilter(getGroupsOfUser(), accessId))" applies, or it is a userid, then this case applies.

Reply:
I don't see a matching test for that.

Reply:
This test already exists: 
AccessIdControllerIntTest::should_returnAccessIdsOfGroupsTheAccessIdIsMemberOf_ifAccessIdOfGroupIsGiven

Reply:
ok :) Missed that. Thanks